src/test/java/org/apache/datasketches/filters/bloomfilter/DirectBitArrayRTest.java

```diff
@@ -32,7 +32,7 @@
 
 public class DirectBitArrayRTest {
 
-  private Memory bitArrayToMemory(HeapBitArray ba) {
+  static private Memory bitArrayToMemory(HeapBitArray ba) {
```

Comment:
Recommended practice is "private static"

Reply:
See [Java language spec 8.4.3](https://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.4.3)

Reply:
fixed